commands/lsp_metals_show_build_target_info.py

```diff
@@ -0,0 +1,37 @@
+from . lsp_metals_text_command import LspMetalsTextCommand
+from . utils import handle_error
+from LSP.plugin.core.protocol import Error
+from LSP.plugin.core.types import List, Any
+
+import sublime
+
+
+class LspMetalsShowBuildTargetInfoCommand(LspMetalsTextCommand):
```

Comment:
The actual command that first list the build targets then invokes the decoding command